Formula/emscripten.rb

```diff
@@ -113,9 +95,8 @@ def install
 
       mkdir llvmpath/"build" do
         system "cmake", "-G", "Unix Makefiles", "..", *args
-        system "make"
-        system "make", "install"
-        system "make", "install-xcode-toolchain" if MacOS::Xcode.installed?
+        system "cmake", "--build", "."
```

Comment:
Any particular reason not to use the `make`/`make install` paradigm  here? In Homebrew we just generally stick with `cmake [...]` followed by `make [install]`, so unless there's a good reason/need for this, I would advise `cmake`/`make` rather than `cmake`/`cmake --build` simply because it's what people are used to seeing around here.

Reply:
Yes, I changed it for consistency with the llvm formula, which uses the same commands for building.

I figured it might make it easier for anyone else trying to maintain this formula to keep using the llvm formula as a reference.

As for why the llvm formula uses this -- it refused to build with just `make [install]` for some reason. (I don't really get why either.)

Reply:
I see, I was not aware that the `llvm` formula also used this. I guess we'll just stick with it then.